### PR TITLE
PACKMP-5 Ability to override the minimal supported version of SonarQube

### PR DESCRIPTION
**Pull request by simonbrandhof**
PACKMP-5 Ability to override the minimal supported version of SonarQube

### CODE REVIEW

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR Add a 'package-info.java' file to document the 'pluginpackaging' package (org.sonarsource.sonar-packaging-maven-plugin:sonar-packaging-maven-plugin:target/generated-sources/plugin/org/sonarsource/pluginpackaging) [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1228)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR Add a 'package-info.java' file to document the 'pluginpackaging' package (org.sonarsource.sonar-packaging-maven-plugin:sonar-packaging-maven-plugin:target/generated-sources/plugin/org/sonarsource/pluginpackaging) [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1228)

File: src/it/overrideQubeMinVersion/pom.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+  <groupId>org.codehaus.sonar</groupId>
+  <artifactId>test</artifactId>
+  <version>1.0</version>
+  <packaging>sonar-plugin</packaging>
+  <name>Package dependencies</name>
+  <description>Package dependencies</description>
+
+  <properties>
+    <sonar.qubeMinVersion>5.1</sonar.qubeMinVersion>
```
Comment (henryju) [inline src/it/overrideQubeMinVersion/pom.xml]: I feel the naming should be more explicit. We could maybe use a specific prefix for packaging plugin 'sonar.packaging'. Something like sonar.packaging.minSupportedSonarQubeVersion. Then we could use sonar.packaging.minSupportedSonarLintVersion for SL

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR Add a 'package-info.java' file to document the 'pluginpackaging' package (org.sonarsource.sonar-packaging-maven-plugin:sonar-packaging-maven-plugin:target/generated-sources/plugin/org/sonarsource/pluginpackaging) [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1228)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR Add a 'package-info.java' file to document the 'pluginpackaging' package (org.sonarsource.sonar-packaging-maven-plugin:sonar-packaging-maven-plugin:target/generated-sources/plugin/org/sonarsource/pluginpackaging) [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1228)

**Comment by henryju** [comment]
LGTM

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR Add a 'package-info.java' file to document the 'pluginpackaging' package (org.sonarsource.sonar-packaging-maven-plugin:sonar-packaging-maven-plugin:target/generated-sources/plugin/org/sonarsource/pluginpackaging) [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1228)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR Add a 'package-info.java' file to document the 'pluginpackaging' package (org.sonarsource.sonar-packaging-maven-plugin:sonar-packaging-maven-plugin:target/generated-sources/plugin/org/sonarsource/pluginpackaging) [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1228)

**Pull request closed without merging**